README.rst

```diff
@@ -258,6 +258,41 @@ To do so set ``uniform=True`` when calling ``extract``, it's false by default fo
 
 NB rdfa structure is not uniformed yet
 
+Returning HTML node
++++++++++++++++++++
+
+It is also possible to get references to HTML node for every extracted metadata item.
+To make it happen, just set the ``return_html_node`` option of ``extract`` method to ``True``.
+As the result, an additional key "nodeHtml" will be included in the result for every
+item. The node is of ``lxml.etree.Element`` type.
+The feature is supported only by syntaxes that are based on HTML: ::
```

Comment:
microformats are also embedded in html, but we don't support returning nodes for them, as it is not implemented. I think it makes more sense to say which syntaxes are supported, explicitly. One may argue that OpenGraph is also based on HTML (annotations are in html meta tags), or even json-ld (annotations are in html, inside a special tag).

Reply:
done :+1: